graphics/macgui/macfontmanager.cpp

```diff
@@ -640,6 +645,65 @@ int MacFontManager::registerFontName(Common::String name, int preferredId) {
 	return id;
 }
 
+int MacFontManager::registerTTFFont(TTFMap *ttfList) {
+	int defaultValue = kMacFontGeneva;
+	int realId = 100;
+	auto checkId = [&](int id) {
+		int i = 0;
+		while (ttfList[i].ttfName != nullptr) {
```

Comment:
If you follow my recommendation, you can simplify this down to `while (ttfList[i].ttfName)`

Reply:
that's true